wasp/src/main/java/com/orhanobut/wasp/MockFactory.java

```diff
@@ -41,18 +42,14 @@ static MockFactory getDefault(Context context) {
     }
 
     @Override
-    public <T> void invokeRequest(WaspRequest waspRequest, CallBack<T> callBack) {
+    public <T> void invokeRequest(WaspRequest waspRequest, CallBack<T> callBack, Parser parser) {
         WaspMock mock = waspRequest.getMock();
         int statusCode = mock.getStatusCode();
 
         if (statusCode < 200 || statusCode > 299) {
-            callBack.onError(new WaspError(
-                    "mock url",
-                    statusCode,
-                    new HashMap<String, String>(),
-                    "Mock test fail",
-                    new byte[0],
-                    0));
+            callBack.onError(new WaspError(parser,
```

Comment:
I would suggest to format it like below in order to separate method params
<pre>
 callBack.onError(new WaspError(
              parser,
              new WaspResponse("mock url", statusCode, Collections.EMPTY_MAP, "Mock test fail", 0, 0), 
              null
));
</pre>